metrics-service/build.gradle

```diff
@@ -0,0 +1,94 @@
+buildscript {
+    repositories mavenRepositories
+    dependencies {
+        classpath("org.springframework.boot:spring-boot-gradle-plugin:${springBootVersion}")
+        classpath("gradle.plugin.com.gorylenko.gradle-git-properties:gradle-git-properties:${gradleGitPropertiesVersion}")
+    }
+}
+
+normalization {
+    runtimeClasspath {
+        ignore("**/*git.properties*")
+        ignore("**/*build-info.properties*")
+    }
+}
+
+apply plugin: 'org.springframework.boot'
+apply plugin: 'io.spring.dependency-management'
+apply plugin: 'com.gorylenko.gradle-git-properties'
+apply from: "$projectDir/gradle/lite.gradle"
+
+springBoot {
+    // This statement tells the Gradle Spring Boot plugin to generate a file
+    // build/resources/main/META-INF/build-info.properties that is picked up by Spring Boot to display
+    // via /info endpoint
+    buildInfo {
+        properties {
+            // Generate extra build info:
+            additionalProperties = [
+                by: System.properties['user.name'],
+                operatingSystem: "${System.properties['os.name']} (${System.properties['os.version']})",
+                number: System.getenv('BUILD_NUMBER') ? System.getenv('BUILD_NUMBER') : "n/a",
+                machine: InetAddress.localHost.hostName
+            ]
+        }
+    }
+}
+
+gitProperties {
+    gitPropertiesDir = new File("${project.rootDir}/${name}/build/resources/main/META-INF")
+}
+
+dependencies {
+    implementation project(':onboarding-enabler-spring')
+    implementation(project(':security-service-client-spring'))
+
+    implementation libraries.springFox
+    implementation libraries.spring_boot_starter_actuator
+    implementation libraries.spring_cloud_starter_eureka
+    implementation libraries.spring_security_web
+    implementation libraries.spring_security_config
+    implementation libraries.gson
+    compileOnly libraries.lombok
+    annotationProcessor libraries.lombok
+}
+
+
+bootJar.archiveName = "${bootJar.baseName}.jar"
+
+jar {
+    enabled true
+}
+
+def uiModule = "${project(':metrics-service-ui').projectDir}"
+
+processResources {
+    dependsOn "copyUi"
+}
+
+task copyUi(type: Copy) {
+    group = "ui"
+
+    dependsOn ":metrics-service-ui:build"
+
+    inputs.dir(uiModule + '/frontend/build')
+    outputs.dir('src/main/resources/static')
+
+    from uiModule + '/frontend/build'
+    into 'src/main/resources/static'
+}
+
+bootRun {
+    if (project.hasProperty('args')) {
+        args project.args.split(',')
+    }
+    systemProperties = System.properties
+}
+
+publishing {
+    publications {
+        mavenJava(MavenPublication) {
```

Comment:
We need to decide whether we will also distinguish between publishing - fat jars and jars as in the other services case. I am fan of providing fat jars.